cmd/tsbs_load_akumuli/creator.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
If I understood correctly, Akumuli does not have the concept of databases and schemas like PostgreSQL or InfluxDB has. And the db creator is not needed to create a database. But the Creator also handles cleaning of old/stale data (check out tsbs_load_timescaledb/creator.go). I would also change it so that the `DBExists` method always returns `true` so the `RemoveOldDb` gets called by tsbs (where you can put the cleanup code). The calling of RemoveOldDb is controlled by the `do-create-db` flag, which by default is true, but if you want to keep the old data you can set it to false

Reply:
Yes, it doesn't have a notion of database/table/schema. And it's not possible to clean up using the database API. You have to run command in terminal to crate or delete the database.